deploy/katello.spec

```diff
@@ -108,7 +100,28 @@ chmod +x %{buildroot}%{homedir}/script/*
 # install man page
 install -m 644 man/katello-service.8 %{buildroot}/%{_mandir}/man8
 
-%post
+%files 
+
+# ------ Common ------------------
+
+%package common
+BuildArch:  noarch
+Summary:    Common runtime components of %{name}
+
+# service-wait dependency
+Requires:       wget
+Requires:       curl
+Requires:       %{?scl_prefix}rubygem-katello
+Requires:       %{name}-installer
+Requires:       rubygem-hammer_cli 
+Requires:       rubygem-hammer_cli_foreman
+Requires:       rubygem-hammer_cli_katello
+Requires:       rubygem-hammer_cli_import
+
+%description common
+Common runtime components of %{name}
+
+%post common
```

Comment:
Would you mind filing an issue to remove this secret token stuff that I don't believe is used anywhere anymore.